CMakeLists.txt

```diff
@@ -110,7 +110,13 @@ INCLUDE_DIRECTORIES(${HDF5_INCLUDE_DIR})
 ADD_SUBDIRECTORY(carta-protobuf)
 ADD_SUBDIRECTORY(carta-scripting-grpc)
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
-INCLUDE_DIRECTORIES(/opt/carta-casacore/include /opt/carta-casacore/include/casacore /opt/carta-casacore/include/casacode)
+option(DevSuppressExternalWarnings "Suppress external warnings (developer use only)" OFF)
+if (DevSuppressExternalWarnings)
+    message(STATUS "disabling warnings from carta-casacore")
+    INCLUDE_DIRECTORIES(SYSTEM /opt/carta-casacore/include /opt/carta-casacore/include/casacore /opt/carta-casacore/include/casacode)
```

Comment:
shall we define this as a variable above, so that we don't need to change this in both places later?

Reply:
Ok, done.